strongbox-distribution/pom.xml

```diff
@@ -355,6 +355,35 @@
                     <tarLongFileMode>gnu</tarLongFileMode>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
+                <version>2.2.0</version>
```

Comment:
Could you please define the plugin's version under the `<pluginManagement/>` section of the `strongbox-parent/pom.xml` and keep it versionless here? Thanks! :)

Reply:
Should I do the same for the appassembler plugin, since I am anyway on it?

Reply:
Yeah, it would probably be a good idea. Thanks!

Reply:
Moved versions for all three plugins!